db/db-api/src/main/java/dao/ActionDAO.java

```diff
@@ -0,0 +1,9 @@
+package dao;
```

Comment:
your package must contain information:
1. type of organization (org)
2. name of organization (my)
3. name of project (adventure)
4. name of mudule (dao)
5. name of artifact (api or impl)
6. and than your packages.